app/routers/traveller/home.py

```diff
@@ -0,0 +1,57 @@
+from typing import List
+
+from fastapi import APIRouter, Depends
+from neomodel import db
+from pydantic import AnyUrl, BaseModel, constr
+
+from ...dependencies.auth import get_registered_user
+from ...helpers.conversion import inflate_query_result
+from ...models import database, validation
+from ...models.database import Package
+
+# from ...helpers.conversion import deflate_request
+
+router = APIRouter()
+
+
+class TopPackagesResponse(BaseModel):
+    id: str
+    name: constr(min_length=5, max_length=100)
+    coverUri: AnyUrl
+    rating: float
+
+
+class TopDestinationResponse(BaseModel):
+    id: str
+    name: constr(min_length=10, max_length=100)
```

Comment:
City name can be 3 letters.

```suggestion
    name: constr(min_length=3, max_length=100)
```